bears/xml/XMLBear.py

```diff
@@ -24,10 +26,11 @@ class XMLBear(LocalBear, Lint):
     gives_corrected = True
 
     def process_output(self, output, filename, file):
-        if self.stdout_output:  # only yield Result if stdout is not empty
-            return self._process_corrected(self.stdout_output, filename, file)
-        if self.stderr_output:  # pragma: no cover
-            return self._process_issues(self.stderr_output, filename)
+        if self.stderr_output:
+            self.use_stderr = True
+        return itertools.chain(
+            self._process_issues(self.stderr_output, filename),
+            self._process_corrected(self.stdout_output, filename, file))
```

Comment:
@SanketDG  I thought it gives stdout also when it gives error .
Because when you validate it against it a schema if invalid it gives output in stdout as well and also errors in stderr
```
<?xml version="1.0"?>
<note xmlns="http://www.w3schools.com" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="note.xsd">
   <from>Jani</from>
   <heading>Reminder</heading>
   <body>Don't forget me this weekend!</body>
</note>
note-invalid.xml:2: element from: Schemas validity error : Element '{http://www.w3schools.com}from': This element is not expected. Expected is ( {http://www.w3schools.com}to ).
note-invalid.xml fails to validate
```